alcs-frontend/src/app/features/board/recon-create-card-dialog/recon-create-card-dialog.component.ts

```diff
@@ -139,17 +161,23 @@ export class ReconCreateCardDialogComponent implements OnInit {
     this.fileNumberControl.reset();
```

Comment:
If you are calling this for every field, I think you can use the formGroup directly and do this.createForm.resetForm()

Reply:
not for every field, only application related once

Reply:
okay